instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -122,7 +122,7 @@ def _instrumented_requests_call(
         method = method.upper()
         span_name = ""
         if name_callback is not None:
-            span_name = name_callback()
+            span_name = name_callback(method, url)
```

Comment:
Just a thought that although this is sufficient for the vast majority of cases, I have a feeling people may want to customize the span name further later on.

For example, one cannot extracted values from the header and add them to the span name.

I don't have a great solution (besides passing in all args via **kwargs, etc), but this may come up as an issue.

Reply:
Yeah I thought of that too. For now, I think this is sufficient and we can change the parameters if the need arises.